data/chaitali-khangar.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Chaitali Khangar",
+  "bio": "I'm a Ruby on Rails and Go lang developer with a passion for mentoring.",
+  "type": "personal",
```

Comment:
We no longer need the `type` key as we only accept personal profiles by default now. Please remove this from your `json` .

```suggestion
```